docs: adjust fswatch overflow message to mention docs with info (#27810)
neovim/neovim: runtime/doc/lua.txt

    vim.api.nvim_command(
      "command! -nargs=1 Watch call luaeval('watch_file(_A)', expand('<args>'))")
<
                                                         *fswatch-limitations*
If you are on Linux and using fswatch, you may need to increase the maximum

> number of watches and queued events as the default limit is pretty low. To

Reviewer: ```suggestion
number of watches and queued events as the default limit can be too low. To
```

> increase the limit, you can run: >sh

Reviewer: Avoid using terms like "you".

```suggestion
increase the limit, run: >sh
```

    sysctl fs.inotify.max_user_watches=100000
    sysctl fs.inotify.max_queued_events=100000
<

> This will increase the limit to 100000 watches and queued events. You can also

Reviewer: ```suggestion
This will increase the limit to 100000 watches and queued events. These lines

> add these lines to /etc/sysctl.conf to make the changes permanent.

Reviewer: ```suggestion
can be added to `/etc/sysctl.conf` to make the changes persistent.
```
